src/content/blog/2021_unitaryhack_results.md

```diff
@@ -12,7 +12,7 @@ tags:
 
 At Unitary Fund, we help those interested in contributing to open quantum projects learn and practice the skills they need to do so. In May 2021, we organized [unitaryHACK](https://unitaryfoundation.github.io/unitaryhack/), a distributed quantum open source hackathon with 4k USD in bounties, physical and digital swag, and 20 participating projects. Working with open source project maintainers, we helped community members with a wide variety of backgrounds make meaningful contributions to their projects. Maintainers as well as Unitary Fund staff were on hand on our [Discord](http://discord.unitary.foundation) to help out with questions about the quantum technologies as well as the tools they use like git, GitHub, and a variety of languages and platforms used in the quantum space (Python, Julia, Q#, C++, Ruby, etc.)
 
-![unitaryHACK digital swag](../images/unitaryHACK-swag.png)
+![unitaryHACK digital swag](/images/unitaryHACK-swag.png)
```

Comment:
Why did the links have to change?

Reply:
One of the Astro versions (I think 3) changed the way link works. Afaiu now all links are absolute with `public` as the default root, as opposed to relative from the place where the link is. A good change, imho.